src/alphabetical.py

```
@@ -0,0 +1,570 @@
+
+def convert_to_title_case(readme_text):
+    # Only title-case bracketed text that is immediately followed by '(' —
+    # this targets the link text portion in Markdown like [Name](url)
+    def _tc_match(m):
+        inner = m.group(1)
+        # Use a robust split that treats common whitespace (including NBSP) as separators
+        tokens = re.split(r"[\s\u00A0]+", inner)
+        cleaned_tokens = []
+        for t in tokens:
+            # strip common punctuation that may be attached to tokens
+            stripped = t.strip("()[]{}.,:;\"'`—–-")
```

Ruff (0.14.14)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)